source/MaterialXContrib/MaterialXMaya/ShadingNodeOverrides.cpp

```diff
@@ -306,10 +308,25 @@ void ShadingNodeOverride<BASE>::updateShader(MHWRender::MShaderInstance& shaderI
     const mx::StringMap& inputs = ogsFragment->getPathInputMap();
     for (const auto& input : inputs)
     {
-        mx::ElementPtr element = document->getDescendant(input.first);
+        const std::string& elementPath = input.first;
+        mx::ElementPtr element = document->getDescendant(elementPath);
         if (!element)
         {
-            continue;
+            std::string nodePath = mx::parentNamePath(elementPath);
```

Comment:
Happens to be I wrote this "promotion" of nodedef input to node input a while back and need it here but
guess it goes with the conversion with Jonathan S that really would be good if "ports" existed all the time.

This resolves finding the nodedef values for texture inputs which don't exist via this mechanism of mapping a MTLX path during code gen and trying to find the value afterwards. Not a big fan of using legacy OGSXML as it's only required here. For all other shader gen the values are on the generated Shader...